### PR TITLE
Dev-migration-DBT-Platform

### PR DESCRIPTION
**Pull request by depsiatwal**

### CODE REVIEW

File: conf/caseworker.py [hunk 1]
```diff
@@ -130,3 +133,24 @@
 
 LITE_FEEDBACK_EMAIL = env.str("LITE_FEEDBACK_EMAIL", "")
 CONFIG_ADMIN_USERS_LIST = env.list("CONFIG_ADMIN_USERS_LIST", default=[])
+
+# Logging formatting
+
+LOGGING = {
```
Comment (depsiatwal) [inline conf/caseworker.py:139]: any reason why this is here when we have it in the base which is backward compatible. 
https://github.com/uktrade/lite-frontend/blob/dev-migration/conf/base.py#L218

File: conf/exporter.py [hunk 2]
```diff
@@ -123,3 +126,24 @@
 ]
 
 SURVEY_URL = env.str("SURVEY_URL", "")
+
+# Logging formatting
+
+LOGGING = {
```
Comment (depsiatwal) [inline conf/exporter.py:132]: any reason why this is here when we have it in the base which is backward compatible. 
https://github.com/uktrade/lite-frontend/blob/dev-migration/conf/base.py#L218

**Comment by depsiatwal** [comment]
Testing on Dev2 and this looks good

**Comment by depsiatwal** [comment]
Testing on Dev2 and this looks good

**Comment by markj0hnst0n** [comment]
Would it be an idea to deploy this somewhere so we can see it's backwards compatible okay?

**Comment by depsiatwal** [comment]
> Would it be an idea to deploy this somewhere so we can see it's backwards compatible okay?

I did test these branches on dev2 all was ok, probably need to retest with most recent changes.

File: conf/base.py [hunk 3]
```diff
@@ -200,9 +202,9 @@
 # AWS
 
 AWS_S3_ENDPOINT_URL = env.str("AWS_S3_ENDPOINT_URL", None)
-AWS_ACCESS_KEY_ID = env.str("AWS_ACCESS_KEY_ID")
-AWS_SECRET_ACCESS_KEY = env.str("AWS_SECRET_ACCESS_KEY")
-AWS_REGION = env.str("AWS_REGION")
+AWS_ACCESS_KEY_ID = env.str("AWS_ACCESS_KEY_ID", None)
+AWS_SECRET_ACCESS_KEY = env.str("AWS_SECRET_ACCESS_KEY", None)
+AWS_REGION = env.str("AWS_REGION", "eu-west-2")
```
Comment (kevincarrogan) [inline conf/base.py:207]: What's the reason this needs to change because of DBT Platform?

**Pull request closed without merging**